src/pool/srv_target.c

```diff
@@ -198,7 +198,7 @@ start_gc_ult(struct ds_pool_child *child)
 	sched_req_attr_init(&attr, SCHED_REQ_GC, &child->spc_uuid);
 	attr.sra_flags = SCHED_REQ_FL_NO_DELAY;
 
-	child->spc_gc_req = sched_create_ult(&attr, gc_ult, child, 0);
+	child->spc_gc_req = sched_create_ult(&attr, gc_ult, child, DSS_DEEP_STACK_SZ);
```

Comment:
It looks to me unlikely a stack overrun issue, I'm wondering if DAOS-11149 is dup of DAOS-11092. Have @mjean308 verified that PR-9667 (from DAOS-11092) doesn't fix DAOS-11149?

Reply:
Well, I am already waiting for @mjean308 to verify if this PR-9924 doesn't fix DAOS-11149 ??!! ;-)

Reply:
I still don't quite see why the unmap call could cause stack overrun (see my comment in ticket), but let's see if this PR could solve the issue.